### PR TITLE
Andrew/login

### PR DESCRIPTION
**Pull request by amillman**
- Add login functionality
- Save user data upon login
- Style main app
- Add more helper classes

### CODE REVIEW

File: components/listings/ListingsScreen.js [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+
+  _handleBackButton() {
+    return true;
+  }
+
   render() {
     return (
       <View>
-        <View>
+        <View style={{paddingTop: 100}}>
```
Comment (nhiquach) [inline components/listings/ListingsScreen.js]: do we need this padding?

File: components/login/LoginScreen.js [hunk 2]
```diff
@@ -1,19 +1,153 @@
...
+        <StatusBar barStyle='light-content' />
+        <KeyboardAvoidingView behavior='padding' style={styles.screen}>
+          <TouchableWithoutFeedback
+            onPress={Keyboard.dismiss}
+          >
+            <View style={styles.contentWrapper}>
+              <Image style={styles.logo} source={require('../../assets/logo-white.png')} />
+              <View><Text style={styles.title}>Replate Marketplace</Text></View>
+              <TextInput
+                style={styles.input}
```
Comment (nhiquach) [inline components/login/LoginScreen.js]: Is it possible to add labels/placeholders for the fields? Email/Password

File: components/login/LoginScreen.js [hunk 3]
```diff
@@ -1,19 +1,153 @@
...
+                onChangeText={(text) => this.setState({password: text})}
+                underlineColorAndroid='transparent'
+                secureTextEntry
+              />
+              <LoadingButton
+                containerStyle={[ComponentStyles.buttonContainer, styles.buttonContainer]}
+                style={[ComponentStyles.buttonText, styles.buttonText]}
+                onPress={this._attemptLogin}
+                isLoading={this.state.isLoading}
+                title="Log in" />
```
Comment (nhiquach) [inline components/login/LoginScreen.js]: nit: "Log In" or "Login"

File: components/listings/ListingsScreen.js (hunk 1 above)
Comment (amillman) [reply components/listings/ListingsScreen.js]: this is dummy data were gonna remove

Comment (nhiquach) [reply components/listings/ListingsScreen.js]: ooh gotcha

**Pull request merged**